packages/aws-cdk-lib/aws-dynamodb/lib/global-table.ts

```diff
@@ -0,0 +1,2152 @@
+     * > specified, the key that the Tablet generates for you will be created with
+     * > default permissions. If you are using CDKv2, these permissions will be
+     * > sufficient to enable the key for use with DynamoDB tables.  If you are
+     * > using CDKv1, make sure the feature flag
+     * > `@aws-cdk/aws-kms:defaultKeyPolicies` is set to `true` in your `cdk.json`.
+     *
+     * @default - The table is encrypted with an encryption key managed by DynamoDB, and you are not charged any fee for using it.
+     */
+    readonly encryption?: TableEncryption;
+
+    /**
+     * External KMS key to use for table encryption.
+     *
+     * This property can only be set if `encryption` is set to `TableEncryption.CUSTOMER_MANAGED`.
+     *
+     * @default - If `encryption` is set to `TableEncryption.CUSTOMER_MANAGED` and this
+     * property is undefined, a new KMS key will be created and associated with this table.
+     * If `encryption` and this property are both undefined, then the table is encrypted with
+     * an encryption key managed by DynamoDB, and you are not charged any fee for using it.
+     */
+    readonly encryptionKey?: kms.IKey;
+
+    /**
+     * The name of TTL attribute.
+     * @default - TTL is disabled
+     */
+    readonly timeToLiveAttribute?: string;
+
+    /**
+     * When an item in the table is modified, StreamViewType determines what information
+     * is written to the stream for this table.
+     *
+     * @default - streams are disabled unless `replicationRegions` is specified
+     */
+    readonly stream?: StreamViewType;
+
+    /**
+     * The removal policy to apply to the DynamoDB Table.
+     *
+     * @default RemovalPolicy.RETAIN
+     */
+    readonly removalPolicy?: RemovalPolicy;
+
+    /**
+     * Regions where replica tables will be created
+     *
+     * @default - no replica tables are created
+     */
+    readonly replicationRegions?: string[];
+
+    /**
+     * The timeout for a table replication operation in a single region.
+     *
+     * @default Duration.minutes(30)
+     */
+    readonly replicationTimeout?: Duration;
+
+    /**
+     * Indicates whether CloudFormation stack waits for replication to finish.
+     * If set to false, the CloudFormation resource will mark the resource as
+     * created and replication will be completed asynchronously. This property is
+     * ignored if replicationRegions property is not set.
+     *
+     * DO NOT UNSET this property if adding/removing multiple replicationRegions
+     * in one deployment, as CloudFormation only supports one region replication
+     * at a time. CDK overcomes this limitation by waiting for replication to
+     * finish before starting new replicationRegion.
+     *
+     * @see https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-dynamodb-globaltable.html#cfn-dynamodb-globaltable-replicas
+     * @default true
+     */
+    readonly waitForReplicationToFinish?: boolean;
```

Comment:
This seems like a holdover from when we used to do the replication ourselves. Is this still necessary?